go/internal/accountutils/utils.go

```diff
@@ -153,19 +163,27 @@ func getOrCreateStorageKey(ks NativeKeystore, keyName string) ([]byte, error) {
 }
 
 func GetOrCreateMasterStorageKey(ks NativeKeystore) ([]byte, error) {
-	return getOrCreateStorageKey(ks, StorageKeyName)
+	return getOrCreateKeystoreKey(ks, StorageKeyName)
 }
 
 func GetOrCreateStorageKeyForAccount(ks NativeKeystore, accountID string) ([]byte, error) {
-	return getOrCreateStorageKey(ks, fmt.Sprintf("%s/%s", StorageKeyName, accountID))
+	return getOrCreateKeystoreKey(ks, fmt.Sprintf("%s/%s", StorageKeyName, accountID))
+}
+
+func GetOrCreateMasterStorageSalt(ks NativeKeystore) ([]byte, error) {
+	return getOrCreateKeystoreKey(ks, StorageSaltName)
+}
+
+func GetOrCreateStorageSaltForAccount(ks NativeKeystore, accountID string) ([]byte, error) {
+	return getOrCreateKeystoreKey(ks, fmt.Sprintf("%s/%s/salt", StorageKeyName, accountID))
```

Comment:
can you update the salt size based on signal's YapDatabase, they have to do same fix has us and it's look like they use a 16bytes salt https://github.com/signalapp/YapDatabase/pull/1/files#diff-550bf64a8af9dd55eca1060a2d9fe142b068e5c01925326e9d22ccb2714febe6R94